accname/name/comp_labelledby.html

```diff
@@ -36,6 +37,93 @@ <h2 id="h2">+ first heading</h2>
   <span id="s1">verify</span><span id="s2">spaces</span><span>FAIL IF INCLUDED</span><span id="s3">between</span><span id="s4">foreach</span>
 </nav>
 
+<h2>Tests for <code>aria-labeledby</code> (non-standard spelling)</h2>
+
+<span id="n1">first label</span>
+<span id="n2">second label</span>
+
+<a href="#" aria-labeledby="n1" class="ex" data-expectedlabel="first label" data-testname="link is labelled by aria-labeledby (non-standard spelling)">x</a>
+<button aria-labeledby="n1" class="ex" data-expectedlabel="first label" data-testname="button is labelled by aria-labeledby (non-standard spelling)">x</button>
+<div role="group" aria-labeledby="n1" class="ex" data-expectedlabel="first label" data-testname="div with role group is labelled by aria-labeledby (non-standard spelling)">x</div>
+
+<h2>Tests for precedence: <code>aria-labelledby</code> vs. <code>aria-labeledby</code> (non-standard spelling)</h2>
+<!-- The "aria-labelledby" spelling should take precedence over "aria-labeledby" for accname -->
+
+<a href="#" aria-labelledby="n1" aria-labeledby="n2" class="ex" data-expectedlabel="first label" data-testname="link labelled by aria-labelledby supercedes aria-labeledby (non-standard spelling)">x</a>
+<button aria-labelledby="n1" aria-labeledby="n2" class="ex" data-expectedlabel="first label" data-testname="button labelled by aria-labelledby supercedes aria-labeledby (non-standard spelling)">x</button>
+<div role="group" aria-labelledby="n1" aria-labeledby="n2" class="ex" data-expectedlabel="first label" data-testname="div labelled by aria-labelledby supercedes aria-labeledby (non-standard spelling)">x</div>
+
+<a href="#" aria-labeledby="n1" aria-labelledby="n2" class="ex" data-expectedlabel="first label" data-testname="link labelled by [aria-labeledby][aria-labelledby] uses aria-labeledby because it is first">x</a>
+<button aria-labeledby="n1" aria-labelledby="n2" class="ex" data-expectedlabel="first label" data-testname="button labelled by [aria-labeledby][aria-labelledby] uses aria-labeledby because it is first">x</button>
+<div role="group" aria-labeledby="n1" aria-labelledby="n2" class="ex" data-expectedlabel="first label" data-testname="div labelled by [aria-labeledby][aria-labelledby] uses aria-labeledby because it is first">x</div>
```

Comment:
```suggestion
<a href="#" aria-labeledby="n1" aria-labelledby="n2" class="ex" data-expectedlabel="second label" data-testname="link labelled by [aria-labeledby][aria-labelledby]">x</a>
<button aria-labeledby="n1" aria-labelledby="n2" class="ex" data-expectedlabel="second label" data-testname="button labelled by [aria-labeledby][aria-labelledby]">x</button>
<div role="group" aria-labeledby="n1" aria-labelledby="n2" class="ex" data-expectedlabel="second label" data-testname="div labelled by [aria-labeledby][aria-labelledby]">x</div>
```
```suggestion
<a href="#" aria-labeledby="n1" aria-labelledby="n2" class="ex" data-expectedlabel="first label" data-testname="link labelled by [aria-labeledby][aria-labelledby] uses aria-labeledby because it is first">x</a>
<button aria-labeledby="n1" aria-labelledby="n2" class="ex" data-expectedlabel="first label" data-testname="button labelled by [aria-labeledby][aria-labelledby] uses aria-labeledby because it is first">x</button>
<div role="group" aria-labeledby="n1" aria-labelledby="n2" class="ex" data-expectedlabel="first label" data-testname="div labelled by [aria-labeledby][aria-labelledby] uses aria-labeledby because it is first">x</div>
```